AzureHosted.md

```diff
@@ -16,9 +32,18 @@ cd scripts/modules
 Import-Module -Name MsIndustryLinks
 ```
 
-## Identify your data source and configure workflow
+### Identify your data source and configure workflow
 
-### Configure data source, transform and sink
+#### Logic App Custom Connectors (API data source/sink only)
+
+A Logic App custom connector is required for an Industry Link to transfer data using an API. It is wrapper around an API to obfuscate the configuration details and enables integration with Logic Apps.
+
+When a custom connector is initially created, it is created as a non-certified custom connector. This only allows your custom connector to be shared within your organization. Your custom connector will need to go through the custom connector certification process if it needs to be made publicly available. Industry Links supports non-certified and certified custom connectors.
+
+Currently, there are no helper cmdlets in [MsIndustryLinks](scripts/modules/MsIndustryLinks/README.md) to configure and create a Logic App custom connector. This may become available as we receive feedback.
```

Comment:
Should probably mention only API key auth is supported. User just needs to provide the swagger.

Reply:
Updated